src/main/c/Makefile

```diff
@@ -105,6 +105,6 @@ libsimdpp-2.0-rc2: libsimdpp-2.0-rc2.tar.gz
 	tar -xzf libsimdpp-2.0-rc2.tar.gz
 
 $(shared_library): libsimdpp-2.0-rc2 ${OBJECTS}
-	mkdir -p $(basename $(shared_library))
+	mkdir -p $(dir $(shared_library))
```

Comment:
My system appears to lack `dir`. Does your system have `dirname`? That seems to be the BSD utility.